controllers/idler/idler_controller.go

```diff
@@ -136,9 +136,14 @@ func (r *Reconciler) ensureIdling(ctx context.Context, idler *toolchainv1alpha1.
 		logger := log.FromContext(ctx)
 		podLogger := logger.WithValues("pod_name", pod.Name, "pod_phase", pod.Status.Phase)
 		if trackedPod := findPodByName(idler, pod.Name); trackedPod != nil {
+			timeoutSeconds := idler.Spec.TimeoutSeconds
+			if isOwnedByVM(pod.ObjectMeta) {
+				// use 1/3rd of the timeout for VMs
+				timeoutSeconds = timeoutSeconds / 3
```

Comment:
Could be even more aggressively I guess... What about x / 12? :) WDYT?

Reply:
Fine with me haha

Reply:
Done fe1ce66c2d352967f07606296a1de72176919d38